drivers/hud_apr/app/models/hud_apr/cell_details_concern.rb

```diff
@@ -0,0 +1,240 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudApr::CellDetailsConcern
+  extend ActiveSupport::Concern
+
+  included do
+    def self.column_headings(question)
+      question_fields(question).map do |key|
+        [key, HudApr::Fy2020::AprClient.detail_headers[key.to_s]]
+      end.to_h
+    end
+
+    def self.question_fields(question)
+      extra_fields = {
+        'Question 5' => age_fields + parenting_fields + veteran_fields + homeless_fields,
+        'Question 6' => pii_fields + universal_data_fields + financial_fields + housing_fields + project_fields + timeliness_fields + inactive_records_fields,
+        'Question 7' => household_fields + parenting_fields + project_fields,
+        'Question 8' => household_fields + parenting_fields + project_fields,
```

Comment:
`extra_fields` should move to a method.
CE Apr will have a different question 9 (so just override `extra_fields` copying in 5-9)
Concern needs to be added to CAPER generators
CE APR will have an additional concern, so it will include this one and then a CE APR specific one that redefines `extra_fields`.

Reply:
✅